server/server.js

```diff
@@ -81,6 +84,39 @@ app.post('/blogs', async (req, res) => {
 
 
 
+//MEMO TO SELF: TWO METHODS SO CHECK WHICH ONE WORKS (OR WORKS BETTER) AS NOT SURE IF I HAVE TO DO A GET (WITH MULTIPLE PARAMS) OR JUST DO A POST REQUEST SO I CAN USE A BODY//
+/*POST method - for acquiring song infos*/
+app.post('/song', async (req, res) => {
+    try { 
+        const { artistName } = req.body;
+        const { trackName } = req.body;
```

Comment:
```suggestion
        const { artistName, trackName } = req.body;
```
You can retrieve multiple things from the object like this! 😁